cmd/podman/pod.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"github.com/urfave/cli"
+)
+
+var (
+	podDescription = `
+   podman pod
+
+   manage pods
+`
+	podCommand = cli.Command{
+		Name:                   "pod",
+		Usage:                  "Manage pods",
+		Description:            podDescription,
+		ArgsUsage:              "CONTAINER-NAME [CONTAINER-NAME ...]",
```

Comment:
I think we should drop this line.  `pod` just holds subcommands, it doesn't have other positional arguments of its own.